yellowbrick/contrib/missing/bar.py

```diff
@@ -0,0 +1,247 @@
+# yellowbrick.contrib.missing.bar
+# Missing Values Bar Visualizer
+#
+# Author:  Nathan Danielsen <nathan.danielsen@gmail.com>
+# Created: Fri Mar 29 5:17:36 2018 -0500
+#
+# Copyright (C) 2018 District Data Labs
+# For license information, see LICENSE.txt
+#
+# ID: bar.py [] nathan.danielsen@gmail.com.com $
+
+"""
+Bar visualizer of missing values by column.
+"""
+
+##########################################################################
+## Imports
+##########################################################################
+
+import numpy as np
+
+from yellowbrick.style.palettes import color_palette
+from .base import MissingDataVisualizer
+
+
+##########################################################################
+## MissingValues Visualizer
+##########################################################################
+
+
+class MissingValuesBar(MissingDataVisualizer):
+    """The MissingValues Bar visualizer creates a bar graph that lists the total
+    count of missing values for each selected feature column.
+
+    When y targets are supplied to fit, the output is a stacked bar chart where
+    each color corresponds to the total NaNs for the feature in that column.
+
+    Parameters
+    ----------
+    alpha : float, default: 0.5
+        A value for bending elments with the background.
+
+    marker : matplotlib marker, default: |
```

Comment:
alpha and marker params left over from Dispersion copy and paste? missing bar width param.